apps/docs/src/components/code/PopupLayout.tsx

```diff
@@ -43,42 +47,53 @@ export const PopupLayout = ({ scope, code }: PopupLayoutProps) => {
   };
 
   return (
-    <section className="relative border-1 -mt-px -ml-px">
-      <HvDialog
-        onClose={() => setIsExpanded(false)}
-        open={isExpanded}
-        fullWidth
-        maxWidth="md"
-      >
-        <HvDialogTitle>Code</HvDialogTitle>
-        <HvDialogContent className="py-sm">
+    <div id={id} className="bg-transparent h-full">
+      <section className="relative border-1 border-t-0 -ml-px h-full">
```

Comment:
is this being used? the `id="1"` samples seem to be gone. 🤔 and if so, it'd be nice if we could merge these

```suggestion
    <section id={id} className="bg-transparent relative border-1 border-t-0 -ml-px h-full">
```